backend/Match/views.py

```diff
@@ -44,5 +44,5 @@ def get(self, request):
 		if not all([match, player_left_side, player_right_side]):
 			return Response({'error': 'Something went wrong when creating the match and players'}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
 		
-		pong_match_url = f'http://localhost:80/pong/{match.id}?token={token.token}'
-		return redirect(pong_match_url)
\ No newline at end of file
+		pong_match_url = f'ws://localhost:8080/pong/{match.id}?token={token.token}'
+		return Response({'match_url': pong_match_url}, status=status.HTTP_200_OK)
```

Comment:
this should be irrelevant like the 'match_url', front end can work itself out on what this is, and we have removed these from other json reponses too!

Reply:
ok, I'll do that, but you'll have to re-review this pr again.